### PR TITLE
add link

### PR DESCRIPTION
**Pull request by KellyBurdine**
Please review this, @casey-collab

[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/1897)

### CODE REVIEW

File: english/bob-barker/barker.md [hunk 1]
```diff
@@ -20,4 +20,6 @@ You hesitate just a moment before replying...
 
 [The red door](2050s/2050s.md)
 
-[Grab some coffee?](../coffee/coffee.md)
\ No newline at end of file
+[Grab some coffee?](../coffee/coffee.md)
+
+[Become a marshmellow](../becomemarshmallow.md)
```
Comment (casey-collab) [inline english/bob-barker/barker.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by casey-collab** [comment]
Unfortunately, this pull request conflicts with another change. Could you merge the master branch of the original repository into your branch, resolve the merge conflicts, and update the pull request?

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
I don't see the conflict, but the broken link comment looks correct. Did you create a new file and then forget to add it?

**Comment by cbuckey-uda** [comment]
Closing this pull request due to lack of activity. If you ever have time to respond to the comments above, feel free to re-open it.

**Pull request closed without merging**